TESTS/mbed_drivers/lp_timeout/main.cpp

```diff
@@ -74,10 +74,13 @@ Case cases[] = {
     Case("1 s delay during deepsleep (attach_us)", test_deepsleep<AttachUSTester<LowPowerTimeout>, 1000000, LONG_DELTA_US>,
          greentea_failure_handler),
 #endif
+    /* Time drifting tests are unstable on CI. Disable them for now. */
+#if TIME_DRIFT_TESTS
 #if !defined(__ARM_FM)  //FastModels not support time drifting test
```

Comment:
we can somehow merge the new macro with this one. because the Fastmodels is in very similar  case

Reply:
Can we remove the fast model ma ro?